src/test/java/org/assertj/core/test/jdk11/ImmutableCollections.java

```diff
@@ -637,8 +637,7 @@ private Object writeReplace() {
       size = input.length; // implicit nullcheck of input
 
       elements = (E[]) new Object[EXPAND_FACTOR * input.length];
-      for (int i = 0; i < input.length; i++) {
-        E e = input[i];
+      for (E e : input) {
```

Comment:
Please revert

Reply:
Oh... I totally overlooked it. Will revert it asap!

Reply:
Reverted